tests/Microsoft.Data.DataFrame.Tests/ArrowIntegrationTests.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Apache.Arrow;
+using Apache.Arrow.Ipc;
+using Xunit;
+
+namespace Microsoft.Data.Tests
+{
+    public class ArrowIntegrationTests
+    {
+        private class ArrayComparer :
```

Comment:
I think it may be easier to read if these arrow comparison methods were split into their own dedicated class. Then they could also be used from multiple tests in the future as well.